libhif/hif-repo.c

```diff
@@ -882,6 +885,34 @@ hif_repo_setup(HifRepo *repo, GError **error)
         return FALSE;
     if (!lr_handle_setopt(priv->repo_handle, error, LRO_GNUPGHOMEDIR, priv->keyring))
         return FALSE;
+
+    if (g_key_file_has_key(priv->keyfile, priv->id, "sslverify", NULL)) {
+        if (g_key_file_get_boolean(priv->keyfile, priv->id, "sslverify", NULL)) {
```

Comment:
Yep, correct. I was emulating what dnf does and completely missed that part:
https://github.com/rpm-software-management/dnf/blob/master/dnf/yum/config.py#L817

Will fix.